src/main/java/controller/AnunciosController.java

```diff
@@ -0,0 +1,22 @@
+package controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(name = "AnunciosController", urlPatterns = {"/AnunciosController"})
+public class AnunciosController extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
indentation

> 

>

Reply:
I didnt get you, I mean, I see everything fine